extensions/notebook/src/dialog/managePackages/installedPackagesTab.ts

```diff
@@ -67,20 +67,42 @@ export class InstalledPackagesTab {
 			this.installedPackagesTable = view.modelBuilder.table()
 				.withProperties({
 					columns: [
-						localize('managePackages.pkgNameColumn', "Name"),
-						localize('managePackages.newPkgVersionColumn', "Version")
+						{
+							value: localize('managePackages.pkgNameColumn', "Name"),
+							type: azdata.ColumnType.text
+						},
+						{
+							value: localize('managePackages.newPkgVersionColumn', "Version"),
+							type: azdata.ColumnType.text
+						},
+						{
+							value: localize('managePackages.deleteColumn', "Delete"),
+							type: azdata.ColumnType.button,
+							options: {
+								icon: {
+									dark: this.dialog.extensionContext.asAbsolutePath('resources/dark/delete_inverse.svg'),
```

Comment:
Having the plumb the extension context through to every corner that needs it for stuff like this is pretty annoying IMO. I'd suggest just doing an IconPathHelper class like https://github.com/microsoft/azuredatastudio/blob/main/extensions/mssql/src/iconHelper.ts#L13

Reply:
@Charles-Gagnon would take another look to the PR. I'm not sure if I addressed all your comments or you're looking for something else. your status shows "requested changes"